packages/@sanity/form-builder/src/FormBuilderContext.ts

```diff
@@ -49,14 +49,17 @@ function memoize(method) {
   }
 }
 
+export interface PatchChannelProps {
```

Comment:
Nice one!

Reply:
I did rename it to `PatchChannelOptions` to match the others